browser/ui/views/profiles/brave_profile_chooser_view.h

```diff
@@ -23,6 +23,10 @@ class BraveProfileChooserView : public ProfileChooserView {
 
   void ResetView() override;
 
+  views::View* CreateDiceSyncErrorView(const AvatarMenu::Item& avatar_item,
```

Comment:
shouldn't we override `CreateCurrentProfileView`?

Reply:
We could, would have to account for Guest profiles as well then.